cfme/tests/services/test_provision_stack.py

```diff
@@ -395,3 +395,78 @@ def test_stack_template_azure():
         upstream: yes
     """
     pass
+
+
+@pytest.mark.meta(blockers=[BZ(1754543, forced_streams=["5.11"])], automates=[1684092])
+@pytest.mark.customer_scenario
+@pytest.mark.tier(2)
+@pytest.mark.provider([EC2Provider], selector=ONE, scope='module')
+def test_retire_catalog_bundle_service_orchestration_item(appliance, request, catalog_item,
+                                                          stack_data):
+    """
+    Bugzilla:
+        1684092
+    Polarion:
+        assignee: nansari
+        startsin: 5.10
+        casecomponent: Services
+        initialEstimate: 1/6h
+        testSteps:
+            1. Add ec2 provider
+            2. Provisioned the catalog bundle with ServiceOrchestration item
+            3. Navigate to My service page
+            4. Retired the bundle
+        expectedResults:
+            1.
+            2.
+            3.
+            4. Catalog bundle should retire with no error
+    """
+    bundle_name = fauxfactory.gen_alphanumeric(15, start="cat_bundle_")
+    catalog_bundle = appliance.collections.catalog_bundles.create(
+        bundle_name, description="catalog_bundle",
+        display_in=True, catalog=catalog_item.catalog,
+        dialog=catalog_item.dialog,
+        catalog_items=[catalog_item.name])
+    request.addfinalizer(catalog_bundle.delete_if_exists)
+
+    # Ordering service catalog bundle
+    service_catalogs = ServiceCatalogs(
+        appliance, catalog_bundle.catalog, catalog_bundle.name, stack_data)
+
+    service_catalogs.order()
+    request_description = (
+        f'Provisioning Service [{catalog_bundle.name}] from [{catalog_bundle.name}]'
+    )
+
+    provision_request = appliance.collections.requests.instantiate(request_description)
```

Comment:
I'm suspect why you need to instantiate this provisioning request? As per my info `order` method itself returning this.

Reply:
@digitronik to get the last message(provisioned service name)  from the service request table. 
 
Last Message
`[EVM] Service [cat_bundle_z69K-20200325-025939] Provisioned Successfully`

Reply:
Made changes as per your suggestion